jenkins/opensearch/benchmark-test-endpoint.jenkinsfile

```diff
@@ -21,20 +21,29 @@ pipeline {
     }
     environment {
         AGENT_LABEL = 'Jenkins-Agent-AL2023-X64-M52xlarge-Benchmark-Test'
+        JOB_NAME = 'benchmark-test'
     }
 parameters {
-        string(
+        password(
             name: 'CLUSTER_ENDPOINT',
             description: 'Provide an endpoint to a cluster for running benchmark tests against it.',
-            trim: true
         )
         booleanParam(
             name: 'SECURITY_ENABLED',
             description: 'Mention if the cluster is secured or insecured.',
             defaultValue: false,
         )
+        password(
```

Comment:
default this to `admin`.

Reply:
I have added that in python end. Can do the same on in Jenkins too.